portfolio/src/main/webapp/script.js

```diff
@@ -42,6 +42,7 @@ function randomizeImage() {
 function onloadFunction() {
     commentFunction('5');
     createMapFunction();
+    console.log("Testing onload()")
```

Comment:
Might want to take out the testing lines if you are done with them?

Reply:
I might leave it in for now because it helps me test when I add changes to this page. I will delete it in once I finish this week's work! Thanks :)